renga/api/repository.py

```diff
@@ -156,8 +157,16 @@ def init_repository(
             if self.git is None:
                 self.git = GitRepo.init(str(path))
         else:
-            assert self.git is None, 'Git repo already exists.'
-            self.git = GitRepo.init(str(path))
+            if self.git is not None:
+                # Cleanup
+                self.renga_path.rmdir()
```

Comment:
I am not sure if it's a good idea to remove the renga path completely.

Reply:
In case of a valid (and working)`renga` repository, If there is an already existing `.renga` directory, then the control never actually reaches this line as [this](https://github.com/SwissDataScienceCenter/renga-python/blob/development/renga/api/repository.py#L152) throws a `FileExistsError` error.
So if the control actually reaches here, then it means that it was created only in the previous `GitRepo.init` call, so I think it should be safe to remove the renga path completely I think ?

Reply:
please see https://github.com/SwissDataScienceCenter/renga-python/pull/137/files#r172632507